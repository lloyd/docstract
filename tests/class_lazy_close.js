/** you don't need to specify @@endclass if you're jumping into
 *  a second class def.  And yeah, this might change if we support
 *  nested classes */

/** @class Foo */
/** @class Bar */
