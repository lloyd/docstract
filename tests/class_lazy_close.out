{
  "classes": [
    {
      "name": "Foo", 
      "source_lines": [
        5, 
        5
      ]
    }, 
    {
      "name": "Bar", 
      "source_lines": [
        6, 
        6
      ]
    }
  ], 
  "desc": "you don't need to specify @endclass if you're jumping into\n a second class def.  And yeah, this might change if we support\n nested classes", 
  "filename": "tests/class_lazy_close.js", 
  "module": "class_lazy_close", 
  "source_lines": [
    1, 
    4
  ]
}
